src/experiments/rl_envs.jl

```diff
@@ -563,8 +568,14 @@ function RLCore.Experiment(
                     critic_loss = agent.policy.learner.critic_loss,
                     entropy_loss = agent.policy.learner.entropy_loss,
                     loss = agent.policy.learner.loss,
-                    reward = total_reward_per_episode.reward[end]
                 )
+                for i in 1:length(env)
+                    if get_terminal(env[i])
+                        @info "training" reward = total_reward_per_episode.rewards[i][end] log_step_increment =
+                            0
+                        break
+                    end
+                end
```

Comment:
Note that if multiple environments reach the terminal at the same time, only one value will be logged.

Reply:
yes.
My idea is,
We can log the reward value of each env individually, such as reward_1, reward_2, ... but this might be redundant.
If I log the reward value multiple times in one step (e.g. reward=x, reward=2*x), then the chart will draw a single line with all the data points, which looks like a zigzag.
I think the envs are identical, the only difference is the random seeds, so which env to log is not important.

Reply:
Agree 👍